sysl2/sysl/swagger/parameters.go

```diff
@@ -54,33 +54,44 @@ func (p Parameters) CookieParams() []Param {
 }
 
 func buildParam(param spec.Parameter, types TypeList, globals Parameters, logger *logrus.Logger) Param {
+
+	fromType := func(t Type) Param {
+		return Param{Field: Field{
+			Name:     param.Name,
+			Optional: !param.Required,
+			Type:     t,
+		},
+			In: param.In,
+		}
+	}
+	fromString := func(typeName string) Param {
+
+		t, found := types.Find(typeName)
+		if !found {
+			logger.Panicf("referenced parameter type %s not found\n", typeName)
```

Comment:
Does this represent a program bug or bad input? Ditto below.

Reply:
bad input.